plugins/org.wso2.developerstudio.eclipse.esb.docker/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Correct the licence header.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/800